src/commands/lambda/functions/instrument.ts

```diff
@@ -283,6 +285,11 @@ export const calculateUpdateRequest = async (
     }
   }
 
+  // Enable ASM
+  if (settings['appsecEnabled'] === true) {
+    newEnvVars[AWS_LAMBDA_EXEC_WRAPPER_VAR] = AWS_LAMBDA_EXEC_WRAPPER
```

Comment:
I think this is onlu needed for .NET, Go, and Java. We should update this.

Reply:
No, the appsec setup for AWS lambda is no longer conditional and you need both env vars no matter the language.

Reply:
Yes this is why indeed.